code/modules/mob/living/simple_animal/abnormality/zayin/hammer_light.dm

```diff
@@ -0,0 +1,298 @@
+//Coded by Coxswain, sprites by Mel, Coxwswain and glowinthedarkmannhandler
+#define STATUS_EFFECT_EVENING_TWILIGHT /datum/status_effect/evening_twilight
+/mob/living/simple_animal/hostile/abnormality/hammer_light
+	name = "Hammer of Light"
+	desc = "A white hammer engraved with yellow runic writing."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "hammer_pedestal"
+	icon_living = "hammer_pedestal"
+	pixel_x = -16
+	base_pixel_x = -16
+	pixel_y = -8
+	base_pixel_y = -8
+	threat_level = ZAYIN_LEVEL
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(70),
+						ABNORMALITY_WORK_INSIGHT = list(70),
+						ABNORMALITY_WORK_ATTACHMENT = list(70),
+						ABNORMALITY_WORK_REPRESSION = list(70)
+						)
+	work_damage_amount = 6
+	work_damage_type = RED_DAMAGE
+	max_boxes = 8
+
+	ego_list = list(
+		/datum/ego_datum/weapon/evening,
+		/datum/ego_datum/armor/evening
+		)
+//	gift_type = /datum/ego_gifts/soda no gift for now.
+	abnormality_origin = ABNORMALITY_ORIGIN_ARTBOOK
+	//add an abnochem at some point. Looking at you, nutter
+
+	pet_bonus = "hums" //saves a few lines of code by allowing funpet() to be called by attack_hand()
+	var/sealed = TRUE
+	var/hammer_present = TRUE
+	var/list/spawned_mobs = list()
+	var/list/banned = list()
+	var/mob/living/carbon/human/current_user = null
+	var/obj/item/ego_weapon/chosen_arms = null
+	var/points
+	var/points_threshold = 150
```

Comment:
I don't believe anything increases this, despite what is written in PR

Reply:
addressed in line 134